.eslintrc

```diff
@@ -1,122 +1,342 @@
 {
-    "rules": {
-        "indent": [
-            2,
-            2
+  "rules": {
+    "quotes": [
+      2,
+      "single"
+    ],
+    "linebreak-style": [
+      2,
+      "unix"
+    ],
+    "semi": [
+      2,
+      "always"
+    ],
+    "no-console": 0,
+    "max-len": [
+      1,
+      140,
+      {
+        "ignoreComments": true
+      }
+    ],
+    "no-cond-assign": [
+      2,
+      "always"
+    ],
+    "no-ex-assign": 2,
+    "curly": 2,
+    "max-depth": [
+      2,
+      5
+    ],
+    "prefer-const": 1,
+    "no-trailing-spaces": [
+      2,
+      {
+        "skipBlankLines": false
+      }
+    ],
+    "one-var": [
+      2,
+      "never"
+    ],
+    "key-spacing": [
+      2,
+      {
+        "beforeColon": false,
+        "afterColon": true
+      }
+    ],
+    "new-cap": 0,
+    "new-parens": 2,
+    "no-mixed-spaces-and-tabs": 2,
+    "no-multiple-empty-lines": [
+      1,
+      {
+        "max": 1
+      }
+    ],
+    "no-new-object": 2,
+    "arrow-spacing": [
+      2,
+      {
+        "before": true,
+        "after": true
+      }
+    ],
+    "operator-assignment": [
+      2,
+      "always"
+    ],
+    "padded-blocks": [
+      2,
+      "never"
+    ],
+    "keyword-spacing": [
+      2,
+      {
+        "before": true,
+        "after": true
+      }
+    ],
+    "space-before-blocks": [
+      2,
+      "always"
+    ],
+    "space-before-function-paren": [
+      2,
+      {
+        "anonymous": "always",
+        "named": "never"
+      }
+    ],
+    "array-bracket-spacing": [
+      2,
+      "never"
+    ],
+    "computed-property-spacing": [
+      2,
+      "never"
+    ],
+    "space-infix-ops": [
+      2,
+      {
+        "int32Hint": true
+      }
+    ],
+    "space-unary-ops": [
+      2,
+      {
+        "words": true,
+        "nonwords": false
+      }
+    ],
+    "no-delete-var": 2,
+    "no-underscore-dangle": 0,
+    "no-shadow-restricted-names": 2,
+    "no-undef-init": 2,
+    "yoda": [
+      2,
+      "never"
```

Comment:
lol... I never noticed we were doing this (probably because eslint was only run on a few .js files we rarely touched)... this is fundamentally at odds with how I like to use `<` over `>` ahaha. (cc @cartant, lol)